Automattic/wp-calypso
# Theme showcase: Don't query wporg on My Themes filter

Related to #79136

## Proposed Changes

The My Themes filter on atomic sites lists the users installed themes, searching on this screen will search the users instealled themes. This change prevents the search from interpolating wporg search results on the same screen.

## Testing Instructions




1. Use the live link
2. Go to /themes?flag=themes/interlaced-dotorg-themes while proxied
3. Switch site to an atomic site
4. Click 'My Themes'
5. Notice that you have a small number of themes listed, the ones you have installed on the site
6. Search for something, e.g. 'omega'
7. You probably get zero results, whereas before this patch you wouldn't
8. Switch to Recommended, which will retain your previous search string
9. Install a community theme like `omega blocks`
10. Go back to My Themes
11. Search for `omega` again
12. Notice you get (probably) just the `omega blocks` themes you installed earlier, and not the `omega` and `omega blocks` themes


## Pre-merge Checklist
